src/client-registry/client-registry-data/client-registry-dialogs/send-patient-to-cr-dialog.component.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Patient } from "../../../types";
+import styles from "./send-patient-to-cr-dialog.scss";
+import {
+  Form,
+  ModalHeader,
+  ModalBody,
+  InlineLoading,
+  ModalFooter,
+  Button,
+} from "@carbon/react";
+import { Payload, submitPatient } from "../../client-registry.resource";
+import {
+  showNotification,
+  showSnackbar,
+  useConfig,
+} from "@openmrs/esm-framework";
+
+interface SendPatientToCRDialogProps {
+  patientUuid: string;
+  patient: Patient;
+  closeModal: () => void;
+}
+
+const SendPatientToCRDialog: React.FC<SendPatientToCRDialogProps> = ({
+  patientUuid,
+  patient,
+  closeModal,
+}) => {
+  const { clientRegistryUrl } = useConfig();
+  const apiUrl = `${clientRegistryUrl}/Patient`;
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(false);
+  const [crPatient, setCRPatient] = useState<Payload>();
```

Comment:
why have this in state if you are not changing it anywhere, maybe you wanna make it just a variable. I see the setter function is unused.